lib/mariaex/row_parser.ex

```diff
@@ -369,6 +369,9 @@ defmodule Mariaex.RowParser do
   defp decode_geometry(<<_len::8-little, srid::32-little, 1::8-little, 3::32-little, num_rings::32-little, rest::bits >>, fields, null_bitfield, acc, datetime, json_library) do
     decode_rings(rest, num_rings, {srid, fields, null_bitfield, acc}, datetime, json_library)
   end
+	defp decode_geometry(<<_len::8-little, _var_a::8-little, _var_b::8-little, srid::32-little, 1::8-little, 3::32-little, num_rings::32-little, rest::bits>>, fields, null_bitfield, acc, datetime, json_library) do
```

Comment:
What are the `_var_a` and `_var_b` values? Are we able to match on these more?

Reply:
I would love to match and maybe do something useful with it, but I have no clue what they are or what they contain. If someone can shed some light on this, I would be happy to adapt the code.